news/2020-02-03-Micro_Profile_OpenTracing_Comes_To_WildFly.adoc

```diff
@@ -0,0 +1,123 @@
+= Micro Profile OpenTracing comes to WildFly =
+ehsavoie
+2020-02-17
+:revdate: 2020-02-03 08:00:00 -0600
+:awestruct-tags: [wildfly, microprofile, configuration, opentracing]
+:awestruct-layout: blog
+:source-highlighter: coderay
+:encoding: utf-8
+:lang: en
+:awestruct-description: How to configure and use Micro Profile OpenTracing within WildFly
+
+Until WildFly 19 you could use MPOT (Micro Profile OpenTracing) to trace your application using environment variables. With wildFly 19 you can now configure several Jaeger Tracers to be used in your applications.
```

Comment:
s/wildFly/WildFly

Reply:
Probably good to say "With WildFly 19 Beta1" as 19 is not out yet.

Reply:
Well I would have wait for WF19 release to publish the article, since wildscribe or docs are not ready yet (in the reference links)

Reply:
and also because it doesn't work with beta1 ;)